articles/flows/guides/single-page-login-flow/add-login-using-single-page-login-flow.md

```diff
@@ -18,7 +18,7 @@ This tutorial will help you add login to your single-page application (SPA) usin
 
 Auth0 makes it easy to implement the single-page login flow by using:
 
-* [Auth0 SPA Quickstarts](/libraries): The easiest way to implement the single-page login flow, which will do most of the heavy-lifting for you. Our [Single-Page App Quickstarts](/quickstart/spa) will walk you through the process.
+* [Auth0.js](/libraries/auth0js): The easiest way to implement the single-page login flow, which will do most of the heavy-lifting for you. Our [Single-Page App Quickstarts](/quickstart/spa) will walk you through the process.
```

Comment:
Will they understand that this filename is a library? It is obvious to us, but will a person moving through this tutorial know that? (Maybe, depending on expected experience level.)

Reply:
We just generalized and called it by its type rather than by the actual file name to match the microsites, which are written for a more general audience. But now we may have removed these references from the microsites, so no problem changing it to auth0.js. I just think that we should be consistent.

Reply:
Yeah, I think that in this specific case, generalized = obfuscated rather than helpful. If we need explanation we could say Auth0.js: This SDK (or "This library") is the easiest...